bigbluebutton-html5/tests/webdriverio/specs/login.spec.js

```diff
@@ -4,18 +4,27 @@ let LandingPage = require('../pageobjects/landing.page');
 let chai = require('chai');
 
 describe('Landing page', function () {
+
+  beforeEach(function () {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000; // default value is 10000
+  });
+
   it('should have correct title', function () {
     LandingPage.open();
-    chai.expect(browser.getTitle()).to.equal(LandingPage.title);
+
+    chai.expect(chromeBrowser.getTitle()).to.equal(LandingPage.title);
+    chai.expect(firefoxBrowser.getTitle()).to.equal(LandingPage.title);
```

Comment:
Good point. I've put every instance of assertions that browsers share into a separate utility class, so there's no unnecessary repetitions anymore: https://github.com/bigbluebutton/bigbluebutton/blob/397a94b1c937cc72b48b6aa96d03126d1ffff3ce/bigbluebutton-html5/tests/webdriverio/utils.js.
It also automatically detects the list of browsers from the config and iterate through it.